tests/components/wyoming/test_satellite.py

```diff
@@ -822,15 +868,19 @@ async def async_pipeline_from_audio_stream(
 
         pipeline_stopped.set()
 
-    with patch(
-        "homeassistant.components.wyoming.data.load_wyoming_info",
-        return_value=SATELLITE_INFO,
-    ), patch(
-        "homeassistant.components.wyoming.satellite.AsyncTcpClient",
-        SatelliteAsyncTcpClient(events),
-    ) as mock_client, patch(
-        "homeassistant.components.wyoming.satellite.assist_pipeline.async_pipeline_from_audio_stream",
-        async_pipeline_from_audio_stream,
+    with (
+        patch(
+            "homeassistant.components.wyoming.data.load_wyoming_info",
+            return_value=SATELLITE_INFO,
+        ),
+        patch(
+            "homeassistant.components.wyoming.satellite.AsyncTcpClient",
+            SatelliteAsyncTcpClient(events),
+        ) as mock_client,
+        patch(
+            "homeassistant.components.wyoming.satellite.assist_pipeline.async_pipeline_from_audio_stream",
```

Comment:
Consider breaking out the patch to a pytest fixture that yields the mock. Then set a side effect on the mock in the test as needed instead. We can avoid repeating all these patches in the tests then.